src/bindings/python/tests/test_onnx/model_zoo_preprocess.ps1

```diff
@@ -0,0 +1,172 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
```

Comment:
can we have cross-platform script?
Can we add usage of this script in the pre-commit because now it is not clear by what tests it is used?
Also, it seems to have much duplication in both added files.

Reply:
I just reflected existing linux-shell script to allow usage on Windows. Linux version is already in pre-commit.

I thought about something python-based in the future, I'll add a ticket.